📚 Doc: Updates to API documentation and README (#3205)
gofiber/fiber: docs/api/redirect.md

toc_max_heading_level: 5
---

> The redirect package is used to redirect the context (request) to a different URL or route.

Reviewer: I think package is not convenient here. Maybe we can replace it to the Redirect methods?

Author: Yeah, this is a bug from ChatGPT 😂 it shouldn't say package

> The redirect package is used to redirect the context (request) to a different URL or route.

Reviewer 2: ```suggestion
The redirect methods are used to redirect the context (request) to a different URL or route.
```
